public/locales/de/liquidity.json

```diff
@@ -0,0 +1,11 @@
+{
+  "liquidityMining.header": "Liq. mining",
+  "liquidityMining.title": "Liquidity mining",
```

Comment:
```suggestion
  "liquidityMining.header": "Liq. Mining",
  "liquidityMining.title": "Liquidity Mining",
```

Reply:
Thanks Jorge, this and similar comments are fixed in [8d38bc1](https://github.com/levelkdev/swapr-dapp/pull/1199/commits/8d38bc14c8f3e95a847d9b09fdb86d1d52c24317)